api/Document.json

```diff
@@ -4659,7 +4659,9 @@
               "prefix": "webkit"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "12",
```

Comment:
I could only find iPadOS 11 and 12.2 to test http://mdn-bcd-collector.appspot.com/tests/api/Document/webkitExitFullscreen, but this is consistent with those results.

https://www.reddit.com/r/apple/comments/b5ketr/psa_ios_122_has_fullscreen_api_custom_html5_video/ talks about it first being in 12, so this seems correct 👍